src/renderer/components/message/Message.tsx

```diff
@@ -819,3 +828,70 @@ function WebxdcMessageContent({ message }: { message: T.Message }) {
     </div>
   )
 }
+
+/**
+ * displays a VCard attachement with avatar, mail & name
+ * onClick imports VCard and creates chat
+ * (only first if multiple contacts are included in VCard)
+ */
+function VCardContent(message: T.Message) {
+  const { selectChat } = useChat()
+  const accountId = selectedAccountId()
+  const { openDialog } = useDialog()
+  const tx = useTranslationFunction()
+  if (message.vcardContact) {
+    const { profileImage, color, displayName, addr } = message.vcardContact
+    const startChatWithContact = async (addr: string) => {
```

Comment:
the logic in this function should be probably wrapped into a try/catch with a log statement and an alert dialog in the catch statement so the user also gets the error

Reply:
maybe we should make a helper function for this at some point, this is also a problem in other places.

Some hook like
```js
function useCallbackWithUserAlert(cb, dependencies, alertMessage) {
const openAlertDialog = useAlertDialog()
return useCallback(()=>{
return async (...args) => {
try {
await cb(...args)
} catch (error) {
log.error(alertMessage, error)
openAlertDialog({message: alertMessage + error && error.message ?? error.toString()})
}
}, [openAlertDialog,...dependencies])
}
```
that we could use in these places:

```js
const startChatWithContact = useCallbackWithUserAlert(async (addr: string) => {}, [])
```

Reply:
transferred this to it's own issue: #3883